### ISSUE TITLE
JOSS Review: Authorship and software paper

### ISSUE THREAD

**Issue by nmstreethran** [LBNL-ETA/MSWH#14]
[JOSS review](https://github.com/openjournals/joss-reviews/issues/2695): comments on authorship and the software paper

## Contribution and authorship

The review criteria asks to check that "the full list of paper authors seems appropriate and complete". However, the list of authors of the software paper (4) does not match the list of contributors in the 'About' section of the README file, `setup.py`, Sphinx HTML documentation, and the DOE CODE submission (5), or the Sphinx LaTeX documentation (6). Also, Hannes Gerhart appears twice in the LaTeX documentation authors list.

### A statement of need

This section clearly explains the problems the software will solve, but does not mention who the target audience is (e.g., homeowners, researchers).

### Quality of writing

The paper could use some work in terms of sentence structure and punctuation. For example, these sentences from line 55:

> The component models we either identified in the existing literature and created a custom Python implementation, or we developed new models. In our implementation we implemented the following existing or new models:

... could be rephrased as: *"The component models were either developed from scratch or implemented in Python based on existing models identified in literature. We implemented the following models:"*.

I can make more specific suggestions if you open a pull request regarding this issue, if you prefer.

Minor suggestions:

- (line 66) 1h -> 1 hour
- (line 72) CA -> California
- (line 81) DOE code -> DOE CODE
- (line 81) master thesis -> master's thesis
- (line 93) Modelica building library `@Wetter:2014` -> Modelica building library **by** `@Wetter:2014`
- (line 97) PC with 12 cores -> computer with a 12-core processor

#### SRCC. (2013, January)

The URL for this reference is no longer available.

#### Blair, N., et al. (2014)

This seems to be a [NREL Technical Report](https://www.nrel.gov/docs/fy14osti/61019.pdf), but the bibliography says it is published in *Renewable Energy*.

**Comment by milicag** [comment]
Thanks @nmstreethran!

Here are some clarifications and the boxes for things I intend to address in either the current or a new PR.

Authorship is correct according to the rules for the JOSS journal after discussion with the team. A person that contributed to a part of code that was removed in the course of time opted to be listed in acknowledgments instead as a co-author.

- [x] I'll make sure to have Hannes appear only once.

- [x] Add target audience to statement of need

I like to keep my style if it is easy to understand the meaning. I don't think you need to rewrite my paper, with all due respect. I appreciate you offer though and if something simply is not clear, please point to it and I am happy to rewrite.

- [x] Address all minor suggestions

- [x] Fix 2 references

**Comment by milicag** [comment]
@nmstreethran: Hannes does not appear two times, I just checked the raw file, he just is associated with multiple affiliations. Can you please clarify the issue? Update: I found the spot and will address it!

**Comment by milicag** [comment]
@nmstreethran Thank you for raising all the issues! I addressed this in PR #12.

**Comment by nmstreethran** [comment]
Thanks for addressing the issues, @milicag. It was not my intention to rewrite your paper; it was just a suggestion and you do not need to go through with it. 'Quality of writing' is one of my review checklist items.

I do believe you need to edit line 55. I think it should be *'The component models were ...'*. I'll have to re-read the paper first and let you know afterwards if anything else is unclear.

~I don't see any changes made to the bibliography file regarding the two references. Everything else looks fine.~ Changes to the bibliography file have been committed directly to the main branch, so it's all good.

**Comment by milicag** [comment]
yes, I felt bad having errors in the bibliography, sorry for not writing on that, there were many details!

I intend to go through and address the remaining issues tonight as needed.

**Comment by milicag** [comment]
Addressed.
